templates/track.php

```diff
@@ -0,0 +1,137 @@
+                <h4>Refund Submitted</h4>
+                <p><?= __('Your refund details have been submitted. You should recieve your refund shortly.', 'blockonomics-bitcoin-payments') ?></p>
+                <h4><i class="material-icons bnomics-alt-icon">autorenew</i></h4>
+                <p><?= __('If you don\'t get refunded in a few hours, contact <a href="mailto:hello@flyp.me">hello@flyp.me</a> with the following uuid:', 'blockonomics-bitcoin-payments') ?><br><span id="alt-uuid">{{altuuid}}</span></p>
+              </div>
+              <!-- REFUNDED with txid-->
+              <div class="bnomics-status-flex bnomics-altcoin-bg-color" ng-show="order.altstatus == 'refunded-txid'" ng-cloak >
+                <h4>Refunded</h4>
+                <h4><i class="material-icons bnomics-alt-icon">autorenew</i></h4>
+                <p><?= __('This payment has been refunded.', 'blockonomics-bitcoin-payments') ?></p>
+                <div>
+                  <?= __('Refund Details:', 'blockonomics-bitcoin-payments') ?>
+                  <div class="bnomics-small bnomics-bold bnomics-left"><?= __('Transaction ID:', 'blockonomics-bitcoin-payments') ?></div> 
+                  <div class="bnomics-small bnomics-left" id="alt-refund-txid">{{order.alttxid}}</div>
+                  <div class="bnomics-small bnomics-bold bnomics-left"><?= __('Transaction URL:', 'blockonomics-bitcoin-payments') ?></div>
+                  <div class="bnomics-small bnomics-left" id="alt-refund-url"><a href="{{order.alturl}}" target="_blank">{{order.alturl}}</a></div>
+                </div>
+              </div>
+              <!-- EXPIRED -->
+              <div class="bnomics-status-flex bnomics-altcoin-bg-color" ng-show="order.altstatus == 'expired'" ng-cloak >
+                <h4>Expired</h4>
+                <h4><i class="material-icons bnomics-alt-icon">timer</i></h4>
+                <p><?= __('Payment Expired. Use the browser back button and try again.', 'blockonomics-bitcoin-payments') ?></p>
+              </div>
+              <!-- LOW/HIGH -->
+              <div class="bnomics-status-flex bnomics-altcoin-bg-color" ng-show="order.altstatus == 'low_high'" ng-cloak >
+                <h4>Error</h4>
+                <h4><i class="material-icons bnomics-alt-icon">error</i></h4>
+                <p><?= __('Order amount too <strong>{{lowhigh}}</strong> for {{order.altsymbol}} payment.', 'blockonomics-bitcoin-payments') ?></p>
+                <p><a href="" ng-click="go_back()"><?= __('Click here', 'blockonomics-bitcoin-payments') ?></a> <?= __('to go back and use BTC to complete the payment.', 'blockonomics-bitcoin-payments') ?></p>
+              </div>
+            </div>
+          <!-- Blockonomics Credit -->
+          <div class="bnomics-powered-by" ng-hide="order.altstatus == 'waiting'"><?= __('Powered by ', 'blockonomics-bitcoin-payments') ?>Blockonomics</div>
```

Comment:
Remove unnecessary code

Reply:
> * Shortcut unnecessary server endpoints that are only hops.
> * Use _$resource.get()_ service instead of _$http_ for cleaner code example https://www.blockonomics.co/js/merchantapp.js
> * Remove _fetch_order_id_. Order Id can get fetched in info_order
> * Altcoin tracking code should use a separate Angular Controller. This way they can still share possibly common code of HTTP $resource endpoints and be a different entity for clarity

*  Shortcuted the unnecessary server endpoints.
*  Included use of  _$resource.get()_
*  Info order has been removed, so fetch order id is still relevant. 
*  Altcoin tracking has been moved to its own separate Controller.